server/repository/src/mock/test_requisition_repository.rs

```diff
@@ -42,3 +43,12 @@ pub fn mock_request_draft_requisition2() -> RequisitionRow {
         r.min_months_of_stock = 0.9;
     })
 }
+
+pub fn mock_request_draft_requisition3() -> RequisitionRow {
```

Comment:
Since push unit test now uses changelogs from database (rather then 'virtual' changelogs defined in test) and changelogs are queries and then translated in the sync push unit tests, needed to make sure that requisition lines that are integrated are linked to store that is active on site (store_b), you'll see similar changes in sync test data